admin/vagrant-acceptance-targets/centos-7/Vagrantfile

```diff
@@ -10,6 +10,30 @@ VAGRANTFILE_API_VERSION = "2"
 
 ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
+$bootstrap_control = <<EOF
```

Comment:
I agree. There is a bit of subtlety around handling empty version, in the acceptance version, though; so I'd rather do that in a separate issue.